contracts/utils/Address.sol

```diff
@@ -34,10 +35,13 @@ library Address {
         if (address(this).balance < amount) {
             revert Errors.InsufficientBalance(address(this).balance, amount);
         }
-
-        (bool success, bytes memory returndata) = recipient.call{value: amount}("");
-        if (!success) {
-            _revert(returndata);
+        if (LowLevelCall.callNoReturn(recipient, amount, "")) {
+            // call successful, nothing to do
+            return;
+        } else if (LowLevelCall.returnDataSize() == 0) {
+            revert Errors.FailedCall();
+        } else {
+            LowLevelCall.bubbleRevert();
```

Comment:
For consistency purposes with other functions only, how about swapping reverts order:
```suggestion
        }
        if (LowLevelCall.returnDataSize() > 0) {
            LowLevelCall.bubbleRevert();
        } else {
            revert Errors.FailedCall();
        }
```